tools/downloader/pytorch_to_onnx.py

```diff
@@ -42,15 +48,16 @@ def parse_args():
     parser.add_argument('--import-module', type=str, default='',
                         help='Name of module, which contains model\'s constructor.'
                         'Requires if model not from Torchvision')
-    parser.add_argument('--input-names', type=str, nargs='+',
+    parser.add_argument('--input-names', type=str, metavar='L[,L...]',
                         help='Space separated names of the input layers')
-    parser.add_argument('--output-names', type=str, nargs='+',
+    parser.add_argument('--output-names', type=str, metavar='L[,L...]',
                         help='Space separated names of the output layers')
-
+    parser.add_argument('--model-params', type=model_parameters, default='',
+                        help='Pairs "name"="value" of model constructor comma-separeted parameters')
     return parser.parse_args()
 
 
-def load_model(model_name, weights, from_torchvision=True, model_path=None, module_name=None):
+def load_model(model_name, weights, from_torchvision=True, model_path=None, module_name=None, model_params=None):
```

Comment:
The default must be `{}`. `**None` isn't valid.

(Although really, I don't think any of these parameters need defaults at all. There is only one call to this function, and it specifies every parameter.)

Reply:
Thanks, good point!